cpp/create_module/algorithm/create.cpp

```diff
@@ -0,0 +1,47 @@
+#include "create.hpp"
+
+const std::unordered_set<mgp::Id> Create::GetIds(const mgp::Value &argument) {
+  std::unordered_set<mgp::Id> result_set;
+  if (argument.IsList()) {
+    for (const auto element : argument.ValueList()) {
+      if (element.IsNode()) {
+        result_set.insert(std::move(element.ValueNode().Id()));
+      } else if (element.IsInt()) {
+        result_set.insert(std::move(mgp::Id::FromInt(element.ValueInt())));
+      } else {
+        throw mgp::ValueException("First argument must be a node, node's id or a list of those.");
+      }
+    }
+  } else if (argument.IsNode()) {
+    result_set.insert(std::move(argument.ValueNode().Id()));
+  } else if (argument.IsInt()) {
+    result_set.insert(std::move(mgp::Id::FromInt(argument.ValueInt())));
+  } else {
+    throw mgp::ValueException("First argument must be a node, node's id or a list of those.");
+  }
+  return result_set;
+}
```

Comment:
You can change this according to other function in PR: https://github.com/memgraph/mage/pull/273